File: HomeWork6_OOP/Core/MVP/ViewConsole.java
package HomeWork6_OOP.Core.MVP;

import java.util.Scanner;

/**
 * Класс ViewConsole имплементирует методы View для работы в консоле
 */
public class ViewConsole implements View {
    Scanner in;

    private String menu = "1: Добавить сотрудника\n2: Список сотрудников\n0: Выход\n";

    private final String successAdded = "\nСотрудник успешно занесен в базу\n";

    public String getSuccessAdded() {
        return successAdded;
    }

    public ViewConsole() {
        in = new Scanner(System.in);
    }

    @Override
    public String inputFullName() {
        System.out.println("Введите ФИО: ");
        return in.nextLine();
    }

    @Override
    public String inputDateOfBirth() {
        System.out.println("Введите дату рождения: ");
        return in.nextLine();
    }

    @Override
    public String inputDivision() {
        System.out.println("Введите Подразделение Организации: ");
        return in.nextLine();
    }

    @Override
    public String inputPosition() {
        System.out.println("Введите должность: ");
        return in.nextLine();
    }

    @Override
    public Integer inputSalary() {
        System.out.println("Введите оклад: ");
        Integer salary = Integer.valueOf(in.nextLine());
        System.out.println(getSuccessAdded());
        return salary;
    }

    @Override
    public void print(String value) {
        System.out.println(value);
    }

    @Override
    public void showMenu() {
        System.out.println(menu);
    }

    @Override
    public String inputMenu() {
        System.out.println("Введите пункт меню: ");
        return in.nextLine();
    }

    @Override
    public void exitMessage() {
        System.out.println("\nВсего хорошего!\n");
    }

    @Override
    public void errorMessage() {
        System.out.println("\nВыбирайте нужный пункт меню!\n");
    }
}

File: HomeWork7_OOP/mvp/View_Console.py
# Класс ViewConsole  для работы в консоле и вывода на экран
class View_Console:
    __start_menu = "1: Добавить сотрудника\n2: Показать сотрудников\n0: Выход"
    __success_added = "Сотрудник успешно добавлен"

    def __init__(self):
        pass

    def show_added_message(self):
        print(f'\n{self.__success_added}\n')

    def show_start_menu(self):
        print(f'\n{self.__start_menu}\n')

    def input_full_name(self):
        return input('Введите ФИО: ')

    def input_date_of_birth(self):
        return input('Введите дату рождения: ')

    def input_division(self):
        return input('Введите подразделение организации: ')

    def input_position(self):
        return input('Введите должность сотрудника: ')

    def input_salary(self):
        return input('Введите оклад: ')

    def input_menu(self):
        return input('Выберите пункт меню: ')

    def show_error_message(self):
        print('\nВыбирайте нужный пункт меню!\n')

    def exit_message(self):
        print('\nВсего доброго!\n')

    def show_workers(self, workers_list):
        for worker in workers_list:
            print('\n')
            for line in worker:
                print(line)
